[2021-03-24 22:36:58.016] [info] Updating Model file from 30517 to latest format...
[2021-03-24 22:36:58.017] [info] Loaded model EF_Mesh_to_Mesh from file Test_Osim_set/visualize_contacts.osim
[2021-03-24 22:36:58.070] [info] Loaded model arm26 from file arm26/arm26_ground_offset.osim
[2021-03-24 22:36:58.515] [error] Error while executing loop step: Traceback (most recent call last):
  File "server.py", line 27, in runLoop
    self.tOS.run_step(action)
  File "/usr/local/nrp/lib/python3.8/site-packages/NRPPythonEngineModule/TOsimLib.py", line 55, in run_step
    self.actuate(action)
  File "/usr/local/nrp/lib/python3.8/site-packages/NRPPythonEngineModule/TOsimLib.py", line 77, in actuate
    func.setValue(float(action[j]))
IndexError: list index out of range

[2021-03-24 22:37:22.710] [info] Updating Model file from 30517 to latest format...
[2021-03-24 22:37:22.711] [info] Loaded model EF_Mesh_to_Mesh from file Test_Osim_set/visualize_contacts.osim
[2021-03-24 22:37:22.760] [info] Loaded model arm26 from file arm26/arm26_ground_offset.osim
[2021-03-24 22:37:23.174] [error] Error while executing loop step: Traceback (most recent call last):
  File "server.py", line 27, in runLoop
    self.tOS.run_step(action)
  File "/usr/local/nrp/lib/python3.8/site-packages/NRPPythonEngineModule/TOsimLib.py", line 55, in run_step
    self.actuate(action)
  File "/usr/local/nrp/lib/python3.8/site-packages/NRPPythonEngineModule/TOsimLib.py", line 77, in actuate
    func.setValue(float(action[j]))
IndexError: list index out of range

[2021-03-24 22:37:39.246] [info] Updating Model file from 30517 to latest format...
[2021-03-24 22:37:39.247] [info] Loaded model EF_Mesh_to_Mesh from file Test_Osim_set/visualize_contacts.osim
[2021-03-24 22:37:39.295] [info] Loaded model arm26 from file arm26/arm26_ground_offset.osim
[2021-03-24 22:37:39.717] [error] Error while executing loop step: Traceback (most recent call last):
  File "server.py", line 27, in runLoop
    self.tOS.run_step(action)
  File "/usr/local/nrp/lib/python3.8/site-packages/NRPPythonEngineModule/TOsimLib.py", line 55, in run_step
    self.actuate(action)
  File "/usr/local/nrp/lib/python3.8/site-packages/NRPPythonEngineModule/TOsimLib.py", line 77, in actuate
    func.setValue(float(action[j]))
IndexError: list index out of range

[2021-03-24 22:38:16.237] [info] Updating Model file from 30517 to latest format...
[2021-03-24 22:38:16.238] [info] Loaded model EF_Mesh_to_Mesh from file Test_Osim_set/visualize_contacts.osim
[2021-03-24 22:39:38.051] [info] Updating Model file from 20303 to latest format...
[2021-03-24 22:39:38.091] [info] Loaded model 3DGaitModel2354 from file Test_Osim_set/std_gait2354_simbody_latest.osim
[2021-03-24 22:39:38.091] [warning] Couldn't find file 'treadmill.vtp'.
[2021-03-24 22:39:38.092] [warning] Couldn't find file 'sacrum.vtp'.
[2021-03-24 22:39:38.092] [warning] Couldn't find file 'pelvis.vtp'.
[2021-03-24 22:39:38.092] [warning] Couldn't find file 'l_pelvis.vtp'.
[2021-03-24 22:39:38.092] [warning] Couldn't find file 'femur.vtp'.
[2021-03-24 22:39:38.092] [warning] Couldn't find file 'tibia.vtp'.
[2021-03-24 22:39:38.092] [warning] Couldn't find file 'fibula.vtp'.
[2021-03-24 22:39:38.092] [warning] Couldn't find file 'talus.vtp'.
[2021-03-24 22:39:38.092] [warning] Couldn't find file 'foot.vtp'.
[2021-03-24 22:39:38.092] [warning] Couldn't find file 'bofoot.vtp'.
[2021-03-24 22:39:38.092] [warning] Couldn't find file 'l_femur.vtp'.
[2021-03-24 22:39:38.092] [warning] Couldn't find file 'l_tibia.vtp'.
[2021-03-24 22:39:38.092] [warning] Couldn't find file 'l_fibula.vtp'.
[2021-03-24 22:39:38.092] [warning] Couldn't find file 'l_talus.vtp'.
[2021-03-24 22:39:38.092] [warning] Couldn't find file 'l_foot.vtp'.
[2021-03-24 22:39:38.092] [warning] Couldn't find file 'l_bofoot.vtp'.
[2021-03-24 22:39:38.092] [warning] Couldn't find file 'hat_spine.vtp'.
[2021-03-24 22:39:38.092] [warning] Couldn't find file 'hat_jaw.vtp'.
[2021-03-24 22:39:38.092] [warning] Couldn't find file 'hat_skull.vtp'.
[2021-03-24 22:39:38.092] [warning] Couldn't find file 'hat_ribs.vtp'.
[2021-03-24 22:39:38.092] [info] ControllerSet 'Controllers' was renamed and is being reset to 'controllerset'.
[2021-03-24 22:39:38.094] [info] ComponentSet 'MiscComponents' was renamed and is being reset to 'componentset'.
[2021-03-24 22:39:38.323] [error] Error while executing loop step: Traceback (most recent call last):
  File "server.py", line 27, in runLoop
    self.tOS.run_step(action)
  File "/usr/local/nrp/lib/python3.8/site-packages/NRPPythonEngineModule/TOsimLib.py", line 55, in run_step
    self.actuate(action)
  File "/usr/local/nrp/lib/python3.8/site-packages/NRPPythonEngineModule/TOsimLib.py", line 77, in actuate
    func.setValue(float(action[j]))
IndexError: list index out of range

[2021-03-24 22:39:56.856] [info] Updating Model file from 20303 to latest format...
[2021-03-24 22:39:56.895] [info] Loaded model 3DGaitModel2354 from file Test_Osim_set/std_gait2354_simbody_latest.osim
[2021-03-24 22:39:56.895] [warning] Couldn't find file 'treadmill.vtp'.
[2021-03-24 22:39:56.896] [warning] Couldn't find file 'sacrum.vtp'.
[2021-03-24 22:39:56.896] [warning] Couldn't find file 'pelvis.vtp'.
[2021-03-24 22:39:56.896] [warning] Couldn't find file 'l_pelvis.vtp'.
[2021-03-24 22:39:56.896] [warning] Couldn't find file 'femur.vtp'.
[2021-03-24 22:39:56.896] [warning] Couldn't find file 'tibia.vtp'.
[2021-03-24 22:39:56.896] [warning] Couldn't find file 'fibula.vtp'.
[2021-03-24 22:39:56.896] [warning] Couldn't find file 'talus.vtp'.
[2021-03-24 22:39:56.896] [warning] Couldn't find file 'foot.vtp'.
[2021-03-24 22:39:56.896] [warning] Couldn't find file 'bofoot.vtp'.
[2021-03-24 22:39:56.896] [warning] Couldn't find file 'l_femur.vtp'.
[2021-03-24 22:39:56.896] [warning] Couldn't find file 'l_tibia.vtp'.
[2021-03-24 22:39:56.896] [warning] Couldn't find file 'l_fibula.vtp'.
[2021-03-24 22:39:56.896] [warning] Couldn't find file 'l_talus.vtp'.
[2021-03-24 22:39:56.896] [warning] Couldn't find file 'l_foot.vtp'.
[2021-03-24 22:39:56.896] [warning] Couldn't find file 'l_bofoot.vtp'.
[2021-03-24 22:39:56.896] [warning] Couldn't find file 'hat_spine.vtp'.
[2021-03-24 22:39:56.896] [warning] Couldn't find file 'hat_jaw.vtp'.
[2021-03-24 22:39:56.896] [warning] Couldn't find file 'hat_skull.vtp'.
[2021-03-24 22:39:56.896] [warning] Couldn't find file 'hat_ribs.vtp'.
[2021-03-24 22:39:56.896] [info] ControllerSet 'Controllers' was renamed and is being reset to 'controllerset'.
[2021-03-24 22:39:56.897] [info] ComponentSet 'MiscComponents' was renamed and is being reset to 'componentset'.
[2021-03-24 22:39:57.149] [error] Error while executing loop step: Traceback (most recent call last):
  File "server.py", line 27, in runLoop
    self.tOS.run_step(action)
  File "/usr/local/nrp/lib/python3.8/site-packages/NRPPythonEngineModule/TOsimLib.py", line 55, in run_step
    self.actuate(action)
  File "/usr/local/nrp/lib/python3.8/site-packages/NRPPythonEngineModule/TOsimLib.py", line 77, in actuate
    func.setValue(float(action[j]))
IndexError: list index out of range

[2021-03-24 22:40:07.961] [info] Updating Model file from 20303 to latest format...
[2021-03-24 22:40:08.001] [info] Loaded model 3DGaitModel2354 from file Test_Osim_set/std_gait2354_simbody_latest.osim
[2021-03-24 22:40:08.002] [warning] Couldn't find file 'treadmill.vtp'.
[2021-03-24 22:40:08.002] [warning] Couldn't find file 'sacrum.vtp'.
[2021-03-24 22:40:08.002] [warning] Couldn't find file 'pelvis.vtp'.
[2021-03-24 22:40:08.002] [warning] Couldn't find file 'l_pelvis.vtp'.
[2021-03-24 22:40:08.002] [warning] Couldn't find file 'femur.vtp'.
[2021-03-24 22:40:08.002] [warning] Couldn't find file 'tibia.vtp'.
[2021-03-24 22:40:08.002] [warning] Couldn't find file 'fibula.vtp'.
[2021-03-24 22:40:08.002] [warning] Couldn't find file 'talus.vtp'.
[2021-03-24 22:40:08.002] [warning] Couldn't find file 'foot.vtp'.
[2021-03-24 22:40:08.002] [warning] Couldn't find file 'bofoot.vtp'.
[2021-03-24 22:40:08.002] [warning] Couldn't find file 'l_femur.vtp'.
[2021-03-24 22:40:08.002] [warning] Couldn't find file 'l_tibia.vtp'.
[2021-03-24 22:40:08.002] [warning] Couldn't find file 'l_fibula.vtp'.
[2021-03-24 22:40:08.002] [warning] Couldn't find file 'l_talus.vtp'.
[2021-03-24 22:40:08.002] [warning] Couldn't find file 'l_foot.vtp'.
[2021-03-24 22:40:08.002] [warning] Couldn't find file 'l_bofoot.vtp'.
[2021-03-24 22:40:08.002] [warning] Couldn't find file 'hat_spine.vtp'.
[2021-03-24 22:40:08.002] [warning] Couldn't find file 'hat_jaw.vtp'.
[2021-03-24 22:40:08.002] [warning] Couldn't find file 'hat_skull.vtp'.
[2021-03-24 22:40:08.002] [warning] Couldn't find file 'hat_ribs.vtp'.
[2021-03-24 22:40:08.002] [info] ControllerSet 'Controllers' was renamed and is being reset to 'controllerset'.
[2021-03-24 22:40:08.004] [info] ComponentSet 'MiscComponents' was renamed and is being reset to 'componentset'.
[2021-03-24 22:40:08.238] [error] Error while executing loop step: Traceback (most recent call last):
  File "server.py", line 27, in runLoop
    self.tOS.run_step(action)
  File "/usr/local/nrp/lib/python3.8/site-packages/NRPPythonEngineModule/TOsimLib.py", line 55, in run_step
    self.actuate(action)
  File "/usr/local/nrp/lib/python3.8/site-packages/NRPPythonEngineModule/TOsimLib.py", line 77, in actuate
    func.setValue(float(action[j]))
IndexError: list index out of range

[2021-03-24 22:40:22.988] [info] Updating Model file from 20303 to latest format...
[2021-03-24 22:40:23.027] [info] Loaded model 3DGaitModel2354 from file Test_Osim_set/std_gait2354_simbody_latest.osim
[2021-03-24 22:40:23.028] [warning] Couldn't find file 'treadmill.vtp'.
[2021-03-24 22:40:23.028] [warning] Couldn't find file 'sacrum.vtp'.
[2021-03-24 22:40:23.028] [warning] Couldn't find file 'pelvis.vtp'.
[2021-03-24 22:40:23.028] [warning] Couldn't find file 'l_pelvis.vtp'.
[2021-03-24 22:40:23.028] [warning] Couldn't find file 'femur.vtp'.
[2021-03-24 22:40:23.028] [warning] Couldn't find file 'tibia.vtp'.
[2021-03-24 22:40:23.028] [warning] Couldn't find file 'fibula.vtp'.
[2021-03-24 22:40:23.028] [warning] Couldn't find file 'talus.vtp'.
[2021-03-24 22:40:23.028] [warning] Couldn't find file 'foot.vtp'.
[2021-03-24 22:40:23.028] [warning] Couldn't find file 'bofoot.vtp'.
[2021-03-24 22:40:23.028] [warning] Couldn't find file 'l_femur.vtp'.
[2021-03-24 22:40:23.028] [warning] Couldn't find file 'l_tibia.vtp'.
[2021-03-24 22:40:23.028] [warning] Couldn't find file 'l_fibula.vtp'.
[2021-03-24 22:40:23.028] [warning] Couldn't find file 'l_talus.vtp'.
[2021-03-24 22:40:23.028] [warning] Couldn't find file 'l_foot.vtp'.
[2021-03-24 22:40:23.028] [warning] Couldn't find file 'l_bofoot.vtp'.
[2021-03-24 22:40:23.028] [warning] Couldn't find file 'hat_spine.vtp'.
[2021-03-24 22:40:23.028] [warning] Couldn't find file 'hat_jaw.vtp'.
[2021-03-24 22:40:23.028] [warning] Couldn't find file 'hat_skull.vtp'.
[2021-03-24 22:40:23.028] [warning] Couldn't find file 'hat_ribs.vtp'.
[2021-03-24 22:40:23.028] [info] ControllerSet 'Controllers' was renamed and is being reset to 'controllerset'.
[2021-03-24 22:40:23.030] [info] ComponentSet 'MiscComponents' was renamed and is being reset to 'componentset'.
[2021-03-24 22:40:23.260] [error] Error while executing loop step: Traceback (most recent call last):
  File "server.py", line 27, in runLoop
    self.tOS.run_step(action)
  File "/usr/local/nrp/lib/python3.8/site-packages/NRPPythonEngineModule/TOsimLib.py", line 55, in run_step
    self.actuate(action)
  File "/usr/local/nrp/lib/python3.8/site-packages/NRPPythonEngineModule/TOsimLib.py", line 77, in actuate
    func.setValue(float(action[j]))
IndexError: list index out of range

[2021-03-24 22:41:30.683] [info] Updating Model file from 20303 to latest format...
[2021-03-24 22:41:30.723] [info] Loaded model 3DGaitModel2354 from file Test_Osim_set/std_gait2354_simbody_latest.osim
[2021-03-24 22:41:30.723] [warning] Couldn't find file 'treadmill.vtp'.
[2021-03-24 22:41:30.723] [warning] Couldn't find file 'sacrum.vtp'.
[2021-03-24 22:41:30.723] [warning] Couldn't find file 'pelvis.vtp'.
[2021-03-24 22:41:30.723] [warning] Couldn't find file 'l_pelvis.vtp'.
[2021-03-24 22:41:30.723] [warning] Couldn't find file 'femur.vtp'.
[2021-03-24 22:41:30.724] [warning] Couldn't find file 'tibia.vtp'.
[2021-03-24 22:41:30.724] [warning] Couldn't find file 'fibula.vtp'.
[2021-03-24 22:41:30.724] [warning] Couldn't find file 'talus.vtp'.
[2021-03-24 22:41:30.724] [warning] Couldn't find file 'foot.vtp'.
[2021-03-24 22:41:30.724] [warning] Couldn't find file 'bofoot.vtp'.
[2021-03-24 22:41:30.724] [warning] Couldn't find file 'l_femur.vtp'.
[2021-03-24 22:41:30.724] [warning] Couldn't find file 'l_tibia.vtp'.
[2021-03-24 22:41:30.724] [warning] Couldn't find file 'l_fibula.vtp'.
[2021-03-24 22:41:30.724] [warning] Couldn't find file 'l_talus.vtp'.
[2021-03-24 22:41:30.724] [warning] Couldn't find file 'l_foot.vtp'.
[2021-03-24 22:41:30.724] [warning] Couldn't find file 'l_bofoot.vtp'.
[2021-03-24 22:41:30.724] [warning] Couldn't find file 'hat_spine.vtp'.
[2021-03-24 22:41:30.724] [warning] Couldn't find file 'hat_jaw.vtp'.
[2021-03-24 22:41:30.724] [warning] Couldn't find file 'hat_skull.vtp'.
[2021-03-24 22:41:30.724] [warning] Couldn't find file 'hat_ribs.vtp'.
[2021-03-24 22:41:30.724] [info] ControllerSet 'Controllers' was renamed and is being reset to 'controllerset'.
[2021-03-24 22:41:30.725] [info] ComponentSet 'MiscComponents' was renamed and is being reset to 'componentset'.
[2021-03-24 22:41:30.955] [error] Error while executing loop step: Traceback (most recent call last):
  File "server.py", line 32, in runLoop
    self.tOS.run_step(action)
  File "/usr/local/nrp/lib/python3.8/site-packages/NRPPythonEngineModule/TOsimLib.py", line 55, in run_step
    self.actuate(action)
  File "/usr/local/nrp/lib/python3.8/site-packages/NRPPythonEngineModule/TOsimLib.py", line 77, in actuate
    func.setValue(float(action[j]))
IndexError: list index out of range

[2021-03-24 22:41:40.286] [info] Updating Model file from 20303 to latest format...
[2021-03-24 22:41:40.325] [info] Loaded model 3DGaitModel2354 from file Test_Osim_set/std_gait2354_simbody_latest.osim
[2021-03-24 22:41:40.325] [warning] Couldn't find file 'treadmill.vtp'.
[2021-03-24 22:41:40.325] [warning] Couldn't find file 'sacrum.vtp'.
[2021-03-24 22:41:40.325] [warning] Couldn't find file 'pelvis.vtp'.
[2021-03-24 22:41:40.325] [warning] Couldn't find file 'l_pelvis.vtp'.
[2021-03-24 22:41:40.325] [warning] Couldn't find file 'femur.vtp'.
[2021-03-24 22:41:40.325] [warning] Couldn't find file 'tibia.vtp'.
[2021-03-24 22:41:40.325] [warning] Couldn't find file 'fibula.vtp'.
[2021-03-24 22:41:40.325] [warning] Couldn't find file 'talus.vtp'.
[2021-03-24 22:41:40.325] [warning] Couldn't find file 'foot.vtp'.
[2021-03-24 22:41:40.325] [warning] Couldn't find file 'bofoot.vtp'.
[2021-03-24 22:41:40.325] [warning] Couldn't find file 'l_femur.vtp'.
[2021-03-24 22:41:40.325] [warning] Couldn't find file 'l_tibia.vtp'.
[2021-03-24 22:41:40.325] [warning] Couldn't find file 'l_fibula.vtp'.
[2021-03-24 22:41:40.326] [warning] Couldn't find file 'l_talus.vtp'.
[2021-03-24 22:41:40.326] [warning] Couldn't find file 'l_foot.vtp'.
[2021-03-24 22:41:40.326] [warning] Couldn't find file 'l_bofoot.vtp'.
[2021-03-24 22:41:40.326] [warning] Couldn't find file 'hat_spine.vtp'.
[2021-03-24 22:41:40.326] [warning] Couldn't find file 'hat_jaw.vtp'.
[2021-03-24 22:41:40.326] [warning] Couldn't find file 'hat_skull.vtp'.
[2021-03-24 22:41:40.326] [warning] Couldn't find file 'hat_ribs.vtp'.
[2021-03-24 22:41:40.326] [info] ControllerSet 'Controllers' was renamed and is being reset to 'controllerset'.
[2021-03-24 22:41:40.327] [info] ComponentSet 'MiscComponents' was renamed and is being reset to 'componentset'.
[2021-03-24 23:32:28.941] [info] Updating Model file from 30517 to latest format...
[2021-03-24 23:32:28.942] [info] Loaded model EF_Mesh_to_Mesh from file Test_Osim_set/visualize_contacts.osim
